Code/frame.js

```diff
@@ -991,6 +993,20 @@ define([
                         if(buttonData.ariaLabel) {
                             $button.attr('aria-label', buttonData.ariaLabel);
                         }
+                        //atribute to display tooltip
+                        $button.attr('data-toggle','tooltip')
+                        //to display tooltip with shortcut
+                        if(buttonData.key){
+                            $button.attr('data-html','true')
```

Comment:
As per our discussion, it was decided to use html for all the tooltips. Irrespective of shortcut keys.

Reply:
This is not done.